app/jobs/render_book_job.rb

```
@@ -0,0 +1,12 @@
+class RenderBookJob < ApplicationJob
+  def perform(book)
+    @book = book
+    @book.update(pending: true)
+    
```

RuboCop (1.69.1)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)